chocolatey/tools-online/chocolateyinstall.ps1

```diff
@@ -16,3 +16,11 @@ $packageArgs = @{
   validExitCodes= @(0, 3010, 1641)
 }
 Install-ChocolateyPackage @packageArgs
+
+$installInfo = @"
+tool: chocolatey
+tool_version: chocolatey-$($env:CHOCOLATEY_VERSION)
+installer_version: $($env:chocolateyPackageVersion)-online
+"@
+
+Out-File -FilePath C:\ProgramData\Datadog\install_info -InputObject $installInfo
```

Comment:
Under which user is this file created? We need it to be readable (owned, probably) by `ddagentuser`.

Reply:
`ddagentuser` has `FullControl` over the `install_info` file but it is not the owner, but `ddagentuser` is not the owner of `datadog.yaml` either. 

For reference here are the owners and access of each of those files on a test run on my testing environment:

```
Path         Owner                  Access                                     
----         -----                  ------                                     
datadog.yaml NT AUTHORITY\SYSTEM    NT AUTHORITY\SYSTEM Allow  FullControl     
                                    BUILTIN\Administrators Allow  FullControl  
                                    BUILTIN\Users Allow  ChangePermissions     
                                    WIN2012R2X64\ddagentuser Allow  FullControl
install_info BUILTIN\Administrators WIN2012R2X64\ddagentuser Allow  FullControl
                                    BUILTIN\Users Allow  ChangePermissions     
                                    BUILTIN\Administrators Allow  FullControl  
                                    NT AUTHORITY\SYSTEM Allow  FullControl     
```

Reply:
👍 Makes sense, since it inherits it from the parent directory.